client/src/pages/activity/Activity.js

```diff
@@ -1,8 +1,5 @@
-import React, { useState, useRef } from "react";
-import { Link } from "react-router-dom";
+import { useState } from "react";
 import "./activity.css";
-import Events from "./events/Events";
```

Comment:
Oh you misunderstand. I meant to clean up the "Activity.js" file itself, since it contains a bunch of deprecated code as a result of separating the Events and Projects code awhile back.

Reply:
I think the file was cleaned. For example, there are only import of useState and activity.css. You may check its older version.

Reply:
Definitely hasn't been cleaned yet. This is present both in this PR and the current version. 
<img width="913" alt="Screen Shot 2022-08-31 at 4 57 27 PM" src="https://user-images.githubusercontent.com/36748732/187805180-e1d39fa7-1fa6-412a-a390-5d583279a5c5.png">

Reply:
I will remove its implementation for being unused, but let's keep the file. I have plan to do with it.

Reply:
Yeah, I never meant to suggest to remove the file, but just to clean it up.